packages/plugin-flex/package.json

```diff
@@ -231,8 +231,11 @@
         "description": "Lists the Flex Releases on the account."
       },
       "flex:plugins:validate": {
-        "description": "Validates the Flex plugin by looking for any issues that could potentially break the plugin when in use. Generates a report with the identified issues, if any."
+        "description": "Validates the Flex plugin by looking for any issues that could potentially break the plugin when in use. Generates a report with the identified issues, if any.",
+        "flags": {
+          "flexui2": "Validates the Flex plugin for compatibility with Flex UI 2.0."
```

Comment:
The flag should be 'flex-ui-2.0' here as well right? Because this shows up in help output for the command.

Reply:
Also, better to not mention 2.0 in the desc since we mean to validate against the latest 2.x version. Can you please change it to - `Validates the Flex plugin for compatibility with the latest Flex UI 2.x version`

Reply:
It is internal will not show up in help output. Will update the description.